ateam-common-packets/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
```

Comment:
I think I'd move the `Cargo.toml` and other rust stuff into the `rust-lib` folder so that the c includes aren't a subfolder of the rust package folder (cause the headers are common and not just for rust)

Reply:
I mean you can change the folder name if you want but I'd probably do this:
* `common-packets/`
  * `README.md`
  * `include/`
    * `stspin.h`
  * `rust-lib/` (or `rust-bindgen/` or whatnot)
    * `Cargo.toml`
    * `Cargo.lock`
    * `build.rs`
    * `lib.rs` (could also put this in a `src/` subfolder if you feel like it)